kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetReaderPredicatePushdownSuite.scala

```diff
@@ -92,20 +92,69 @@ class ParquetReaderPredicatePushdownSuite extends AnyFunSuite
 
   private def generateRowsGroup(rowGroupIdx: Int): Seq[Row] = {
     def values(rowId: Int): Seq[Any] = {
+      // One of the columns in each row group is all null depending on the [[rowGroupIdx]] and
```

Comment:
trailing `and`, is that for this sentence or the next line? The next line starts with capital `T` so not sure